src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -133,14 +132,21 @@ public void InitializeService(ServiceHost serviceHost)
         {
             try
             {
+                ConnectionInfo connInfo;
+                ScriptingService.ConnectionServiceInstance.TryFindConnection(
+                    parameters.ConnectionString,
```

Comment:
>parameters.ConnectionString [](start = 20, length = 27)

this should be looking up connections via a URI.  Are we overloading connection string parameter?  I'd prefer to add an other parameter for owneruri and make it optional

Reply:
The function does take an ownerUri, and in this case its the connectionString. Is that what you asked, or did I not understand your question?